okhttp/build.gradle.kts

```diff
@@ -210,6 +211,82 @@ android {
   }
 }
 
+// From https://github.com/Kotlin/kotlinx-atomicfu/blob/master/atomicfu/build.gradle.kts
+val compileJavaModuleInfo by tasks.registering(JavaCompile::class) {
```

Comment:
Please refactor this so it’s the same `applyJavaModules()` call we have in all of our other modules? Having two copy-pastes of this configuration will confuse me for all time

Reply:
(You could definitely have a parameter to toggle the options that only the okhttp module needs? Why doesn’t this module use `me.champeau.mrjar`?!)